docs/docs/resources/faq.md

```diff
@@ -54,7 +54,7 @@ Gives you
 | ---- | ----- | ------ |  ----- | 
 | Example | 7 hours	| 18 hours| 	9 hours, 30 minutes | 
 
-**But storing a Inline DQL in a field comes with a limitation**: While the value that gets displayed in the result is the calculated one, **the saved value inside the Inline DQL is still your calculation**. This means you cannot filter for the Inlines' result like:
+**But storing a Inline DQL in a field comes with a limitation**: While the value that gets displayed in the result is the calculated one, **the saved value inside your metadata field is still your Inline DQL calculation**. The value literally is `= this.end - this.start - this.pause`. This means you cannot filter for the Inlines' result like:
```

Comment:
[...] The value is literally [...]

instead of literally is. "Literally is" also works but needs a comma to separate out "literally" I think. We're getting into extreme pedantry here so sorry for that.